packages/tupaia-web/src/views/LandingPage/SingleProjectLandingPage.tsx

```diff
@@ -100,6 +101,7 @@ export function SingleProjectLandingPage({
       {accessType && (
         <ActionLink
           variant="contained"
+          target= {`${useLandingPage()}`}
```

Comment:
I'm not sure what this is doing - `useLandingPage` should be returning an object of data, loading state etc. Instead, it would be more ideal if we could just set `target='_blank'` on the link